generate/generate.go

```diff
@@ -117,13 +121,52 @@ type Generator struct {
 // It is fine to pass the plugin.CodeGeneratorRequest to every protoc generator
 // unaltered; this is what protoc does when calling out to the generators and
 // the generators should already handle the case where they have nothing to do.
-func (g *Generator) GeneratePkg(path string) error {
+func (g *Generator) GeneratePkg(path string, gens []config.Generator) error {
 	req := g.requestForPkg(path)
+	// Run generator for go
```

Comment:
I imagined that `go` would be listed too, even if it's almost always going to be on, or even if we end up statically linking with it.

Is there a reason not to? If you left this for a later PR, just leave a TODO for clarity.

Reply:
This is currently an issue with the ini parser, it doesn't handle multiple sections with the same name and getting the key values from the sections correctly. I need to submit a PR to handle the multiple sections to the ini package. However, this should likely be changed to use the library rather than shelling out, I'll just add that as a TODO.